src/jsMain/kotlin/moe/nea89/website/KConsole.kt

```diff
@@ -142,7 +145,14 @@ class KConsole(
             return
         }
         val command = parts[0]
-        println("Running command: $command")
+        wholecommand = ""
+        println("whole command:")
+        for (element in parts){
+            wholecommand += element + " "
+        }
+        println(wholecommand)
```

Comment:
`wholecommand` is just the same as `commandLine`, so you should use that instead (because that also respects `"`)